Custom-Video-Driver/Lets-Build-OTPublisher/Single-Cam-Capturer/TBExampleVideoCapture.m

```diff
@@ -345,18 +338,18 @@ - (void) setCaptureSessionPreset:(NSString*)preset {
         if ([session canSetSessionPreset:preset] &&
             ![preset isEqualToString:session.sessionPreset]) {
             
-            [_captureSession beginConfiguration];
-            _captureSession.sessionPreset = preset;
-            _capturePreset = preset;
+            [self.captureSession beginConfiguration];
+            self.captureSession.sessionPreset = preset;
+            self->_capturePreset = preset;
```

Comment:
@jvalli this one required the arrow. I am not why since is it declared in the same place as the `caputureSession`